src/common/gui/SurgeGUIEditor.h

```diff
@@ -126,6 +130,12 @@ class SurgeGUIEditor : public EditorType,
 
    virtual Steinberg::tresult PLUGIN_API onSize(Steinberg::ViewRect* newSize) override;
    virtual Steinberg::tresult PLUGIN_API checkSizeConstraint(Steinberg::ViewRect* newSize) override;
+   virtual Steinberg::tresult PLUGIN_API setContentScaleFactor(ScaleFactor factor) override
+   {
+      scaleFactor = factor;
```

Comment:
It will be left for future refactoring to properly use this scaleFactor, if that's even possible. Now this is just used to resolve wrong GUI size from host on initial open vs. reopen. More info here: https://github.com/surge-synthesizer/surge/issues/1951